p0

```diff
@@ -13,4 +13,8 @@ process.emit = function (name, data, ...args) {
   return originalEmit.apply(process, arguments);
 };
 
-require(`${__dirname}/dist/index.js`).main();
+if (process.env.NODE_ENV === "dev") {
```

Comment:
See the previous comment: https://github.com/p0-security/p0cli/pull/102#discussion_r1757690405

Specifically, running `NODE_ENV=dev p0` should work. I think this breaks that.

Reply:
I can change it to something else. NODE_DBEUG = true for example.

Reply:
Reverted